### PR TITLE
[test-only]check that image is open

### PR DESCRIPTION
**Pull request by ScharfViktor**
related https://github.com/owncloud/ocis/issues/8602

we had test for this bug but unfortunately the test didn't expect an error

the image viewer was open, but the image did not appearScreenshot 2024-03-07 at 17 39 25

I'm wait locators now
image

we can check the response, but sometimes that doesn't give us a 100% guarantee that everything works. Example, openning broken pdf file: 
image

for txt or md files we can check content(ctr+c and evaluate clickboard) but it dificult when user viewer

### CODE REVIEW

**Comment by ScharfViktor** [comment]
don't merge please before fixing https://github.com/owncloud/ocis/issues/8602

File: tests/e2e/support/objects/app-files/utils/editor.ts
Comment (kulmann) [inline tests/e2e/support/objects/app-files/utils/editor.ts:7]:
```suggestion
const imageViewer = '.stage'
```

The `stage` selector will only appear if loading the content was successful.

Comment (ScharfViktor) [reply tests/e2e/support/objects/app-files/utils/editor.ts:7]: thanks 😍

**Review by SagarGi — COMMENTED** [review]
Test failing as the selector was changed. LGTM :+1: if CI is happy.

**Review by saw-jan — APPROVED** [review]

**Review by nabim777 — APPROVED** [review]

**Pull request merged**
